arq/cli.py

```diff
@@ -22,7 +22,9 @@
 
 
 @click.command('arq')
-@click.version_option(VERSION, '-V', '--version', prog_name='arq')
+@click.version_option(
+    importlib.metadata.version('arq'), '-V', '--version', prog_name='arq',
```

Comment:
Possible `importlib.metadata.PackageNotFoundError`. It's better to write the helper and fallback to unknown.